From e9d394a46ec5447ee61595bd87480cbd1d62bd4b Mon Sep 17 00:00:00 2001
From: "Guoniu.Zhou" <guoniu.zhou@nxp.com>
Date: Wed, 14 Mar 2018 19:01:01 +0800
Subject: [PATCH 3512/5242] MLK-17804-3: dts: change gpio pin for qm ov5640
 sensor

commit  b1445b1b0a3bc3c059838f90f13c9df58555f1e3 from
https://source.codeaurora.org/external/imx/linux-imx.git

Because ov5640 need power and rest pin, so replace mipi
csi gpio with generic gpio controller pins.

Reviewed-by: sandor.yu <sand.yu@nxp.com>
Signed-off-by: Guoniu.Zhou <guoniu.zhou@nxp.com>
(cherry picked from commit b2bca258493d10028beb127480c58e66400465fb)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8qm-mek-mipi-csi2.dts     |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-mipi-csi2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-mipi-csi2.dts
index 812fcae..0f2f4fd 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-mipi-csi2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-mipi-csi2.dts
@@ -52,8 +52,8 @@
 		clocks = <&clk IMX8QM_CLK_DUMMY>;
 		clock-names = "csi_mclk";
 		csi_id = <0>;
-		pwn-gpios = <&gpio0_mipi_csi0 1 GPIO_ACTIVE_LOW>;
-		rst-gpios = <&gpio0_mipi_csi0 0 GPIO_ACTIVE_HIGH>;
+		pwn-gpios = <&gpio1 28 GPIO_ACTIVE_LOW>;
+		rst-gpios = <&gpio1 27 GPIO_ACTIVE_HIGH>;
 		mclk = <24000000>;
 		mclk_source = <0>;
 		mipi_csi;
-- 
1.7.9.5

